man/man1/pkcstok_admin.1.in

```diff
@@ -0,0 +1,192 @@
+.TH pkcstok_admin 1 "July 2024" "@PACKAGE_VERSION@" "openCryptoki"
+.SH NAME
+pkcstok_admin \- utility to administrate token directories of the Opencryptoki
+token repository.
+.
+.
+.
+.SH SYNOPSIS
+.
+.B pkcstok_admin
+.I command
+.RB [ OPTIONS ]
+.PP
+.
+.B pkcstok_admin
+.BR \-\-help | \-h
+.br
+.B pkcstok_admin
+.BR \-\-version | \-v
+.
+.
+.
+.SH DESCRIPTION
+.
+The \fBpkcstok_admin\fP utility can be used to administrate token directories of
+the Opencryptoki token repository with proper permissions and owners.
+By default token directories are owned by the \fB@pkcs_group@\fP group, and
+every user that is a member of the \fB@pkcs_group@\fP group has access to the
+token and its token objects (i.e. keys, certificates, etc.).
+.PP
+A token directory can alternatively be owned by a token specific group.
+Specify the group name with keyword \fBusergroup\fP in the slot definition in
+the \fBopencryptoki.conf\fP configuration file locate in
+\fB@sysconfdir@/opencryptoki/\fP.
+With that, only users that are members of the configured token specific group
+have access to the token and its token objects. Users that are only members of
+the \fB@pkcs_group@\fP group, but are not also a members of the token specific
```

Comment:
a member of

Reply:
change to 'but are not also members of the' because 'users' is plural.